src/GlobalStyles.js

```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+* {
+        box-sizing: border-box;
+		margin: 0;
+		padding: 0;
+		font-family: "Montserrat", Arial, sans-serif;
```

Comment:
Currently Montserrat is only affecting your layout, when the user has it installed on his computer. You need to import it or link to the font to make sure, everybody has access to the font.

You can link the font via the <link> tag at the <head> in the index.html file, however your storybook won't display your components using the font, as it can't access it (I tried myself).
Alternatively, as @import doesn't work in GlobalStyles (as far as i know), you can create a "font.css" file, import the font there and import at the top of this GlobalStyles.js:
```
import './fonts.css'
```
That workaround did the trick for me; if you stumble across a better solution, please let me know! :)